src/index.js

```diff
@@ -32,7 +32,15 @@ function processFeedUrls(lines) {
 async function enqueueFeedJobs(feedJobs) {
   feedJobs.forEach(async feedJob => {
     console.log(`Enqueuing Job - ${feedJob.url}`);
-    await feedQueue.add(feedJob);
+    await feedQueue.add(feedJob, {
+      attempts: 8,
```

Comment:
These magic numbers (`8`, `60 * 1000`) should probably go in the env.  Would be good in another PR.